{
	"id": "oval:com.redhat.cve:def:202439614",
	"version": "636",
	"class": "vulnerability",
	"metadata": {
		"title": "python-django: Potential denial-of-service in django.utils.translation.get_supported_language_variant() (moderate)",
		"reference": [
			{
				"ref_id": "CVE-2024-39614",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2024-39614",
				"source": "CVE"
			}
		],
		"description": "DOCUMENTATION: A vulnerability was found in Python-Django in the get_supported_language_variant() function. The issue triggers when parsed with very long strings, including a specific set of characters, leading to a potential denial of service attack.",
		"advisory": {
			"from": "secalert@redhat.com",
			"severity": "Moderate",
			"updated": {
				"date": "2024-09-17"
			},
			"cve": [
				{
					"text": "CVE-2024-39614",
					"cvss3": "5.3/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L",
					"cwe": "CWE-1287",
					"href": "https://access.redhat.com/security/cve/CVE-2024-39614",
					"impact": "moderate",
					"public": "20240709:1400"
				}
			],
			"affected": {
				"resolution": [
					{
						"state": "Affected",
						"component": [
							"python-django20",
							"python-django20-bash-completion",
							"python3-django20"
						]
					}
				]
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/a:redhat:openstack:16.1",
					"cpe:/a:redhat:openstack:16.1::el8"
				]
			},
			"issued": {}
		},
		"affected": {}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:202023064005",
										"comment": "python3-django20 is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:202023064006",
										"comment": "python3-django20 is signed with Red Hat redhatrelease2 key"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:202023064001",
										"comment": "python-django20-bash-completion is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:202023064002",
										"comment": "python-django20-bash-completion is signed with Red Hat redhatrelease2 key"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.cve:tst:202023064003",
										"comment": "python-django20 is installed"
									},
									{
										"test_ref": "oval:com.redhat.cve:tst:202023064004",
										"comment": "python-django20 is signed with Red Hat redhatrelease2 key"
									}
								]
							}
						]
					}
				],
				"criterions": [
					{
						"test_ref": "oval:com.redhat.cve:tst:20178761011",
						"comment": "Red Hat Enterprise Linux 8 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:com.redhat.cve:tst:20178761012",
				"comment": "Red Hat Enterprise Linux must be installed"
			}
		]
	}
}
